debug.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/*
+ * Plugin Name: Debug Loader
+ * Description: A set of debug tools and plugins
+ */
+
+
+/**
+ * Test Jquery Updates
+ *
+ * There are multiple versions per WordpressVersion the loader tries to pick the most recent one
+ */
+if ( ( defined( 'VIP_ENABLE_TEST_JQUERY_UPDATES' ) && VIP_ENABLE_TEST_JQUERY_UPDATES ) ) {
```

Comment:
We can add the querystring check here to dynamically load as needed:

```suggestion
$has_test_jquery_param = isset( $_GET['vip_load_test_jquery_updates'] );
$has_test_jquery_constant = defined( 'VIP_ENABLE_TEST_JQUERY_UPDATES' ) && true === VIP_ENABLE_TEST_JQUERY_UPDATES;
if ( $has_test_jquery_param || $has_test_jquery_constant ) {
```

Reply:
Sweet, thanks I haven't look into how to access the request, but this seems super simple:) (not sure what I was worried about). I will test it locally and with CLI (where there is no request).